src/main/proto/wfa/measurement/internal/kingdom/protocol_data.proto

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 The Cross-Media Measurement Authors
```

Comment:
Maybe file should be "protocol_metadata.proto"?

Reply:
done